apps/meteor/client/views/admin/subscription/components/cards/ActiveSessionsPeakCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Skeleton } from '@rocket.chat/fuselage';
+import type { ReactElement } from 'react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { useFormatDate } from '../../../../../hooks/useFormatDate';
+import { useStatistics } from '../../../../hooks/useStatistics';
+import type { CardProps } from '../FeatureUsageCard';
+import FeatureUsageCard from '../FeatureUsageCard';
+
+const ActiveSessionsPeakCard = (): ReactElement => {
+	const { t } = useTranslation();
+	const { data, isLoading } = useStatistics();
+	const formatDate = useFormatDate();
+
+	const { maxMonthlyPeakConnections } = data || {};
+
+	const total = 200;
+	const used = maxMonthlyPeakConnections || 0;
+
+	const exceedLimit = used >= total;
+
+	const card: CardProps = {
+		title: t('ActiveSessionsPeak'),
+		infoText: t('ActiveSessionsPeak_InfoText'),
+		showUpgradeButton: exceedLimit,
+	};
+
+	return (
+		<FeatureUsageCard card={card}>
+			{!isLoading && maxMonthlyPeakConnections ? (
```

Comment:
Copying Hugo's comment
```
I know that some people prefer this way, but following our recommended best practices, this is not recommended...
https://alexkondov.com/tao-of-react/#conditional-rendering

Reply:
I didnt get it. this is exactly what the blog post suggests 
<img width="313" alt="image" src="https://github.com/RocketChat/Rocket.Chat/assets/5263975/7b2754b9-b5a9-45a9-af3b-2e575c0593e6">
`condition ? if : else`

Reply:
Ouch, didn't see the `?` there, forget it